### ISSUE TITLE
All photos from all years downloaded when Timeline/2021 is path

### ISSUE THREAD

**Issue by mabahj** [jotta/jotta-cli-issues#130]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
```
-------------------------------------------
jottad executable : /usr/bin/jottad
jottad appdata    : /var/lib/jottad
jottad version    : 0.9.38023
jotta-cli version : 0.9.38023
-------------------------------------------
```
**Description of problem:**
```jotta-cli download Photos/Timeline/2021 .``` downloads every single photo stored on my Jotta account. ```jotta-cli download Photos/Timeline/2021/1 .``` works as intended

**Expected:**
```jotta-cli download Photos/Timeline/2021 .``` should download only photos from 2021.

**jotta-cli status (`jotta-cli status`):**
```
N/A
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
The log contains a lot of these. The paths there also seem very wrong.
```
pid:1163 2021/01/25 21:20:21 Attempting download of /photos/Timeline/2021
pid:1163 2021/01/25 21:20:22 Download starting: /photos/Timeline/2021 (isfile:false)=> /media/files/photo/JottaSync/tmp/2021
pid:1163 2021/01/25 21:20:23 Duplicate name for /photos/Timeline/2021/2005/4/IMG_1195.JPG => /photos/Timeline/2021/2005/4/IMG_1195_2.JPG 
pid:1163 2021/01/25 21:20:23 Duplicate name for /photos/Timeline/2021/2005/4/IMG_1201.JPG => /photos/Timeline/2021/2005/4/IMG_1201_2.JPG 
pid:1163 2021/01/25 21:20:23 Duplicate name for /photos/Timeline/2021/2005/4/IMG_1211.JPG => /photos/Timeline/2021/2005/4/IMG_1211_2.JPG 
pid:1163 2021/01/25 21:20:24 Duplicate name for /photos/Timeline/2021/2005/4/Video005.3gp => /photos/Timeline/2021/2005/4/Video005_2.3gp 
pid:1163 2021/01/25 21:20:25 Duplicate name for /photos/Timeline/2021/2006/5/P1010001.JPG => /photos/Timeline/2021/2006/5/P1010001_2.JPG 

```

**Traceback**
```

```

**Additional info:**
* Below I first start to download Photos/Timeline/2021/1, which is 108 files.
* I then abort this and delete the files.
* I list the content of 2021, which of course only contains one month
* I then download the entire 2021. I abort the scanning after 5300 files have been found.
```
[zsh|0.90]% jotta-cli download Photos/Timeline/2021/1 .
> starting download Photos/Timeline/2021/1 => /media/files/photo/JottaSync/tmp/1
> gathering metadata ... (ctrl-C to cancel)
> 108 files (668.89MiB) found in 1.011708957s       
started download 9b74f9e7ba
  run 'jotta-cli download --abort=9b74f9e7ba' to cancel
  run 'jotta-cli observe --downloadid=9b74f9e7ba' to view progress
  or see 'jotta-cli list downloads' for information about 
  all current, queued or failed downloads
OK
[zsh|0.71]% jotta-cli download --abort=9b74f9e7ba
OK
[zsh|0.99]% rm -rf 1
[zsh|0.91]% jotta-cli ls Photos/Timeline/2021
1 
[zsh|1.10]% jotta-cli download Photos/Timeline/2021 .
> starting download Photos/Timeline/2021 => /media/files/photo/JottaSync/tmp/2021
> gathering metadata ... (ctrl-C to cancel)
> 5300 files (13.88GiB) found                       ^C
aborted...%                                                                                                                                                                                                [zsh|1.02]% 
```

**Comment by Kimbsen** [comment]
Well get that fixed for the next release.  ```jotta-cli download Photos/Timeline/2021 .``` should of course only download photos from 2021 :)

**Comment by Kimbsen** [comment]
We've fixed this in version 0.9.41047 ( unstable ).

It will be in the next stable release as well. Not sure when that will be though.

**Comment by Kimbsen** [comment]
this should be fixed in the 0.9.41303 ( stable ) also now.
